### DISCUSSION TITLE
What is the difference between debug and release?

### DISCUSSION THREAD

**Discussion by KhazanGu** [HomeSpan/HomeSpan#208] [General]
I designed and finished a 5V led circuit using the esp32 module. I exported the 3.3V, TX, RX, GND ports from PCB and connect them to my USB to TTL hub, The hub connect to my computer. It works well after I flashed code into the PCB and reboot.

But the status led kept flashing after I cut down the USB to TTL hub and used a 5V power supply to provide the PCB. It doesn't respond after I press the control button.

**Comment by HomeSpan** [comment]
It sounds like the issue  is related to the way the serial port is wired on your board.  See issues #125 and #131 for details and potential solutions.   I'm not sure I understand your question about "debug" vs. "release".

**Reply by KhazanGu** [reply]
Thanks a lot. Thanks a lot. I just wonder if add other configurations if I disconnect the TX and RX ports.
I solved the problem after referencing your suggestion.
